scripts/utils/release-utils.ts

```diff
@@ -39,7 +40,7 @@ export const isValidVersionInput = (input: string): boolean =>
 /**
  * Determines if the provided `version` is a semver pre-release or not
  * @param version the version string to evaluate
- * @retuns true if the `version` is a pre-release, false otherwise
+ * @returns {boolean} true if the `version` is a pre-release, false otherwise
```

Comment:
<img width="1086" alt="截屏2022-11-08 上午9 35 22" src="https://user-images.githubusercontent.com/57180744/200452343-2f299418-19a6-40fe-be08-4ef98918b5f7.png">
there will be error info because of this config 
`  'jsdoc/require-returns-check': ['error'],`

Reply:
I think the errors for this file (`release-utils.ts`) are only a result of a typo in `@returns` - right now it's missing the second "r". I think if we restore it, that should fix it